docs/gds-supported-platform/making-things-private-in-harbor.md

```diff
@@ -0,0 +1,64 @@
+Making images private in Harbor
+===============================
+
+Sometimes, we need to make our Docker images private. By default, the GSP Harbor
+is publicly accessible. Here are instructions on how to make images
+private.
+
+1. Add the following to your build-pipeline.yml:
+
+```
+  harbor_source: &harbor_source
+    username: ((harbor.harbor_username))
+    password: ((harbor.harbor_password))
+    harbor:
+      url: ((harbor.harbor_url))
+      prevent_vul: "false"
+      public: "false"
+    notary:
+      url: ((harbor.notary_url))
+      root_key: ((harbor.root_key))
+      delegate_key: ((harbor.ci_key))
+      passphrase:
+        root: ((harbor.notary_root_passphrase))
+        snapshot: ((harbor.notary_snapshot_passphrase))
+        targets: ((harbor.notary_targets_passphrase))
+        delegation: ((harbor.notary_delegation_passphrase))
+
+  resource_types:
+  - name: harbor
+    type: docker-image
+    privileged: true
+    source:
+      repository: govsvc/gsp-harbor-docker-image-resource
+      tag: "0.0.1553882420"
+
+  resources:
+  - name: my-image
+    type: harbor
+    source:
+      <<: *harbor_source
+      repository: registry.((cluster.domain))/my-collection/my-image
+```
+
+The crucial line is `public: false` to ensure that anonymous users (ie
```

Comment:
```suggestion
The crucial line is `public: "false"` to ensure that anonymous users (ie
```
The content here should match the content above

Reply:
Good shout. This boolean NEEDS to be a string, otherwise the API won't accept it.

Reply:
It does now. :flushed:

Reply:
We found this out the hard way, IIRC.